pkg/prowgen/prowgen_tests.go

```diff
@@ -107,6 +108,20 @@ func DiscoverTests(r Repository, openShiftVersion string, cronOverride *string)
 			cfg.Tests = append(cfg.Tests, *cronTestConfiguration)
 		}
 
+		securityTestConfiguration := cioperatorapi.TestStepConfiguration{
+			As:       "security",
+			Optional: true,
+			MultiStageTestConfiguration: &cioperatorapi.MultiStageTestConfiguration{
+				Environment: cioperatorapi.TestEnvironment{
+					"PROJECT_NAME": r.Repo,
+					"ORG_NAME":     r.Org,
+				},
+				Workflow: pointer.String("openshift-ci-security"),
+			},
+		}
+
+		cfg.Tests = append(cfg.Tests, securityTestConfiguration)
```

Comment:
This will add one test per openshift version and per branch, I was wondering if we can that option one level above

in Prowgen config something like this:
```golang
			if isFirstVersion {
				isFirstVersion = false
				options = append(options, withNamePromotion(r, branchName))

				options = append(options, WithSnykScanning(r, branchName))
			} else {
				options = append(options, withTagPromotion(r, branchName))
			}
```

Also, how does it distinguish between branches?